net/acme/files/run.sh

```diff
@@ -132,7 +132,6 @@ post_checks()
     if [ -f "$STATE_DIR/post-acme.user" ]; then
         log "Running user-provided cleanup script from $STATE_DIR/post-acme.user."
         . "$STATE_DIR/post-acme.user" || return 1
```

Comment:
This means that the cleanup below will be aborted if the user script exits with an error. That is probably not a good idea (for post - it's fine for pre!). So I think it's better to move this until after the built-in cleanup - that way we can still propagate the return code, but we make sure that the script cleans up after itself.

Reply:
Consider it done.